presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -37,10 +37,16 @@
      * Pre-requisites for the tests in this class:
      *
      * (1) hive.properties file should have this property set: hive.security=sql-standard
-     * (2) tempto-configuration.yaml file should have definitions for two connections to Presto server:
-     * "alice@presto" that has "jdbc_user: alice" and
-     * "bob@presto" that has "jdbc_user: bob"
-     * (all other values of the connection are same as that of the default "presto" connection).
+     * (2) tempto-configuration.yaml file should have definitions for the following connections to Presto server:
+     *          - "alice@presto" that has "jdbc_user: alice"
+     *          - "bob@presto" that has "jdbc_user: bob"
+     *          - "hive@presto" that has "jdbc_user: hive"
+     *     (all other values of the connection are same as that of the default "presto" connection).
+     * (3) hive-site.xml should have the following property added:
+     *     <property>
+     *       <name>hive.users.in.admin.role</name>
+     *       <value>hive</value>
```

Comment:
This property is already present in docker images. I added a comment, just to make the pre-requisites clear.